### ISSUE TITLE
insert function's signature?

### ISSUE THREAD

**Issue by hainm** [serge-sans-paille/pythran#500]
from matmut.py example

```python
In [1]: import matmul as mm

In [2]: mm.matrix_multiply?
Type: builtin_function_or_method
```

vs python's pure mode

```python
In [2]: mm.matrix_multiply?
Signature: mm.matrix_multiply(m0, m1)
Docstring: <no docstring>
```

what do you think? (Cython insert the signature a while ago).

**Comment by serge-sans-paille** [comment]
That's seem reasonnable; `Will do'' ;-)

**Comment by serge-sans-paille** [comment]
@hainm just to be sure, when you mean signature, you mean ``foo(x,y)`` or ``foo(float64[], float64[])`` ?

**Comment by hainm** [comment]
I meant `foo(x, y)`, but `foo(float64[], float64[])` will be much better (good to have type hint-like)

**Comment by serge-sans-paille** [comment]
Should be fixed in #506

**Comment by serge-sans-paille** [comment]
Fixed by 4b43f5daca6af64c2e8d2e1e57ae2a3ec11d8001
